movies-manager/src/containers/main/main.component.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import "./main.component.scss";
+import { Wrapper } from "../wrapper/wrapper.component";
+import { ResultFilter, ResultSort, MovieCard, IMovieInfo } from "../../components";
+
+interface IMovie extends IMovieInfo {
+    id: number;
+};
+
+const movies = require('../../data.json');
+const blockName = 'result';
+
+export const Main = () => {
+    const allMoviesGenres: string[] = movies.reduce((allGenres: string[], { genres }: IMovie) => {
+        allGenres.push(...genres);
+        return allGenres;
+    }, ['All']);
+    const moviesGenres = Array.from(new Set(allMoviesGenres));
+    const moviesSortSet = [
+        { 
+            title: 'voteAverage',
+            isActive: true,
+        },
+        { 
+            title: 'voteCount',
+            isActive: false,
+        },
+        { 
+            title: 'releaseDate',
+            isActive: false,
+        },
+        { 
+            title: 'revenue',
+            isActive: false,
+        },
+    ];
+const moviesCards = movies.map(({id, ...rest}: IMovie) => {
```

Comment:
> All const should be aligned by one vertical line. Setup code formatter to format your code files on every save of commit.

will improve in [Tasl-5] with reason to avoid duplicate improvements